src/modules/reviewrecords.js

```diff
@@ -130,12 +152,13 @@ class ReviewRecords {
         let tasksToAwait = [this.waitForRemotePin(multihash)];
         if (previousVersionMultihash) {
             // This is a review update
-            tasksToAwait.push(this.setForwardPointerForReviewRecord(previousVersionMultihash, multihash));
+            tasksToAwait.push(this.setForwardPointerForReviewRecord(previousVersionMultihash, multihash, reviewRecord));
```

Comment:
Seems like we only have one task to wait for now, do you want to simplify this with a single await?

Reply:
there is the `waitForRemotePin` task at line 152 which is always needed and the then the `setForwardPointerForReviewRecord` which is necessary only if the RR is an update. Maybe you missed the first one?

Reply:
Oh yeah, I misread that part. Thanks for clarifying.